frontend/src/layout/navigation-3000/components/KeyboardShortcut.tsx

```diff
@@ -1,28 +1,44 @@
 import { isMac } from 'lib/utils'
 import { HotKeyOrModifier } from '~/types'
 import './KeyboardShortcut.scss'
+import clsx from 'clsx'
 
-type KeyboardShortcutProps = Partial<Record<HotKeyOrModifier, true>>
+const IS_MAC = isMac()
+const KEY_TO_SYMBOL: Partial<Record<HotKeyOrModifier, string>> = {
+    shift: '⇧',
+    command: IS_MAC ? '⌘' : 'ctrl',
+    option: IS_MAC ? '⌥' : 'alt',
+    arrowup: '↑',
+    arrowdown: '↓',
+    arrowleft: '←',
+    arrowright: '→',
+    enter: '↩',
+    escape: 'esc',
+    tab: '⇥',
+    space: '␣',
+}
+/** For consistency, we always show modifiers in this order, before other keys. */
+const MODIFIER_PRIORITY: HotKeyOrModifier[] = ['shift', 'command', 'option']
+
+interface KeyboardShortcutProps extends Partial<Record<HotKeyOrModifier, true>> {
+    /** Whether this shortcut should be shown with muted opacity. */
+    muted?: boolean
+}
 
-export function KeyboardShortcut(props: KeyboardShortcutProps): JSX.Element {
-    const useMacSymbols = isMac()
+export function KeyboardShortcut({ muted, ...keys }: KeyboardShortcutProps): JSX.Element {
+    const sortedKeys = Object.keys(keys).sort(
+        (a, b) =>
+            (MODIFIER_PRIORITY.indexOf(a as HotKeyOrModifier) || 0) -
+            (MODIFIER_PRIORITY.indexOf(b as HotKeyOrModifier) || 0)
+    ) as HotKeyOrModifier[]
```

Comment:
This seems to have reversed the shortcut order: 
<img width="263" alt="Screenshot 2023-04-26 at 09 26 25" src="https://user-images.githubusercontent.com/1851359/234501444-497e47d8-3a2c-4e92-8705-02876c3cddb0.png">

Reply:
Ah, of course, `.sort()` is ascending